index.js

```diff
@@ -166,6 +167,13 @@ function start() { // eslint-disable-line max-statements
     // Set the X-XSS-Protection HTTP header as a basic protection against XSS
     app.use(helmet.xssFilter());
 
+    // Allow cross-origin resource sharing
+    app.use('/v1/bots', cors());
+    app.use('/v1/botActions', cors());
+    app.use('/v1/botData', cors());
+    app.use('/v1/events', cors());
+    app.use('/v1/rooms', cors());
+
```

Comment:
What about adding moving these endpoints to the config.? May be,  put them in an array and loop through the array here. Something like how "ip whitelisting" is done. This will enable the Admin to add the endpoints without making a code change.